app/src/interfaces/input-block-editor/input-block-editor.vue

```diff
@@ -28,6 +30,8 @@ const props = withDefaults(
 	},
 );
 
+const bus = useBus();
```

Comment:
I used a unified place because I needed to put same signature on both `input-block-editor.vue` and `plugins.ts`.
I didn't want to repeat code so it felt the best way to do it, but let me know if I should change it.

Reply:
Thats a fair reason :D